test/test_helpers/location_test_helper.rb

```diff
@@ -21,7 +22,7 @@ module LocationTestHelper
 
   FORMATTED_GEOSEARCH_RESPONSE = [
     {
-      "name" => "University of California, San Francisco, Parnassus Avenue, Inner Sunset, San Francisco, San Francisco City and County, California, 94131, USA",
+      "name" => "University of California, San Francisco, San Francisco, San Francisco City and County, California, USA",
```

Comment:
it would be nice to  see more test cases, exercising more code paths

Reply:
added two more test cases 👍

Reply:
💙